.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI for Lablink Services
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - "lablink-client-base/lablink-client-service/**"
+      - "lablink-allocator/lablink-allocator-service/**"
+      - ".github/workflows/ci.yml"
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  lint:
+    name: Lint - ${{ matrix.package }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - package: lablink-client-service
+            python-version: "3.10"
```

Comment:
what constrains the python versions? python version differs in tests. let's decide on one